contentcuration/contentcuration/frontend/channelList/views/ChannelSet/ChannelSelectionList.vue

```diff
@@ -1,56 +1,42 @@
 <template>
 
-  <VContainer fluid>
+  <VContainer fluid class="pa-0 pb-5">
     <template v-if="loading">
       <VProgressLinear
         indeterminate
         color="primary"
       />
-      <p class="headline mb-0">
+      <p class="headline mb-0 mt-5">
         {{ $tr('loading') }}
       </p>
     </template>
-    <p v-else-if="listChannels && !listChannels.length" class="headline mb-0">
+    <p v-else-if="listChannels && !listChannels.length" class="headline mb-0 mt-4">
       {{ $tr('noChannelsFound') }}
     </p>
     <template v-else>
       <VTextField
```

Comment:
This needs to be outside of the v-if/v-else dealing with the channels - as soon as you type in something that doesn't match any channels, the text box disappears, and there doesn't seem to be a way to clear the search term.